war/src/main/webapp/scripts/yui/animation/animation-debug.js

```diff
@@ -540,7 +540,7 @@ YAHOO.util.AnimMgr = new function() {
      * Adds an animation instance to the animation queue.
      * All animation instances must be registered in order to animate.
      * @method registerElement
-     * @param {object} tween The Anim instance to be be registered
+     * @param {object} tween The Anim instance to be registered
```

Comment:
I think that this should remain as a duplicated word because it is in a JavaScript library that the Jenkins project has forked from its upstream.  The change is not necessary here and will make future comparison with upstream more difficult.
```suggestion
     * @param {object} tween The Anim instance to be be registered
```